Indaba_2022_Prac_Template.ipynb

```diff
@@ -6,7 +6,7 @@
         "id": "m2s4kN_QPQVe"
```

Comment:
<p>For the form, is it possible to removed the name and email required fields? I.e., can this be taken from the Google account that the user has logged in with? If the participants are going to be doing multiple quizzes, then it might be annoying to keep entering those details. Also, is it possible to allow them to reubmit their answers if they want to try again?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
So I amde the form editable so people can retry. Unfortunelty, it appears that google forms always makes manually enter in your email address and can't get it automatically (maybe some content thing).